### ISSUE TITLE
 Remove Non-functional Close Button from Download Popup

### ISSUE THREAD

**Issue by sudhanshubsr** [Sls0n/Prismify#14]
![000236-31](https://github.com/user-attachments/assets/faeb6456-6259-455e-833c-fc3fbefcc5ad)

The Close Button on the download popup is non-functional. The popup appears only the first time after the page renders, and subsequent attempts show a toast directing the user to log in before they can download the image.

To avoid confusion, I suggest removing the Close Button entirely. This would streamline the user experience and prevent users from mistakenly thinking they can close the popup.

Steps to Reproduce:

Render the page.
Attempt to download an image.
Observe that the Close Button on the popup does not work.
Expected Behavior:
Either the Close Button should function correctly, or it should be removed to prevent unnecessary confusion.

Actual Behavior:
The Close Button is non-functional, and the popup only appears once after the page renders.

Proposed Solution:
Remove the Close Button from the download popup.

**Comment by viiccwen** [comment]
Hello, @sudhanshubsr.
I can't reproduce this issue (the message box doesn't pop out)
It seems already fixed, right?

**Comment by Sls0n** [comment]
> Hello, [@sudhanshubsr](https://github.com/sudhanshubsr). I can't reproduce this issue (the message box doesn't pop out) It seems already fixed, right?

Yes I removed the download cap so this dialog component is futile as of now
